autosubmit/platforms/platform.py

```diff
@@ -97,7 +97,7 @@ class CopyQueue(Queue):
     A queue that copies the object gathered.
     """
 
-    def __init__(self, maxsize: int = -1, block: bool = True, timeout: float = None, ctx: Any = None) -> None:
+    def __init__(self, maxsize: int = -1, block: bool = True, timeout: float = 0.0, ctx: Any = None) -> None:
```

Comment:
Interesting. We declare a `self.timeout` on `CopyQueue`, but I cannot find where it's used.

Looking inside `Queue` (parent class, from Python stdlib), I also cannot see where `self.timeout` gets used for the `Queue` object -- it only receives timeouts via function args.

You can see that the `CopyQueue.put` function, for instance, receives a timeout. That may be completely different than this `self.timeout` you set the type for here. My guess is that this `self.timeout` can be removed, but probably for a 4.1.17 or later version. 🫠